qrcode/main.go

```diff
@@ -41,12 +43,25 @@ Usage:
 	if *size <= 0 {
 		checkError(fmt.Errorf("Error: value of -s should > 0"))
 	}
-	if len(flag.Args()) == 0 {
-		flag.Usage()
-		checkError(fmt.Errorf("Error: no content given"))
-	}
 
 	content := strings.Join(flag.Args(), " ")
+	if *inFile != "" {
```

Comment:
Hey,

I think reading content from STDIN/a file is a nice addition :)

After this pull request there are three ways to specify content:

- String from input file:

`$ qrcode -i input_filename | display`

- String on command line:

`$ qrcode hello | display`

- String on STDIN:

`$ echo -n hello | qrcode | display`

- No arguments: (same as string on STDIN, only you type into the terminal and then press Ctrl+D)

`$ qrcode | display`

If you try the "qrcode hello | display" method, it now hangs and waits for something on STDIN.

I suggest taking content from one source only (first the -i flag, then command line arguments, or lastly STDIN), so the usage is more obvious and easier to explain. This would work well.


thanks,

Tom